packages/account-v2/src/pages/TwoFactorAuthentication/TwoFactorAuthenticationDisabled.tsx

```diff
@@ -0,0 +1,91 @@
+            {children}
+        </a>
+    );
+};
+
+export const TwoFactorAuthenticationDisabled = () => {
+    const { data: authorizeData } = useAuthorize();
+    const { email = '' } = authorizeData ?? {};
+    const { data: secretKeyData, isLoading, mutate } = useTwoFactorAuthentication();
+    const { secretKey } = secretKeyData ?? {};
+    useEffect(() => {
+        mutate({ totp_action: 'generate' });
+    }, [mutate]);
+
+    return (
+        <div className='flex flex-1 w-full h-full sm:flex-col-reverse sm:col-span-2'>
+            <section className='py-0 px-[14px]'>
+                <Text
+                    align='start'
+                    as='h2'
+                    className='ml-[12px] mb-[15px] text-[16px] text-system-light-prominent-text leading-normal sm:mr-1.2rem'
+                    weight='bold'
+                >
+                    How to set up 2FA for your Deriv account
+                </Text>
+                <Timeline className='pt-0 px-[14px] pb-[16px] m-[12px] w-full' lineHeight='xs'>
+                    <Timeline.Item
+                        itemTitle={
+                            <Text align='start' className='leading-tight text-[14px] text-system-light-prominent-text'>
+                                Scan the QR code below with your 2FA app. We recommend{' '}
+                                <TwoFactorAuthenticationLink
+                                    aria-label='Authy' // TODO: Remember to localize this
+                                    href={TWO_FACTOR_AUTHENTICATION_URLS.authy}
+                                >
+                                    Authy
+                                </TwoFactorAuthenticationLink>{' '}
+                                or{' '}
+                                <TwoFactorAuthenticationLink
+                                    aria-label='Google Authenticator' // TODO: Remember to localize this
+                                    href={TWO_FACTOR_AUTHENTICATION_URLS.googleAuthenticator}
+                                >
+                                    Google Authenticator
+                                </TwoFactorAuthenticationLink>
+                                .
+                            </Text>
+                        }
+                    >
+                        <div className='flex flex-col items-center mb-[16px]'>
+                            {isLoading ? (
+                                <Loader isFullScreen={false} />
+                            ) : (
+                                <TwoFactorAuthenticationQRCode emailAddress={email} secretKey={secretKey} />
+                            )}
+                        </div>
+                    </Timeline.Item>
+                    <Timeline.Item
+                        itemTitle={
+                            <Text
+                                align='start'
+                                className='m-w-[500px] text-[14px] leading-tight text-system-light-prominent-text'
```

Comment:
The `Text` component uses size property which can make changes on the font size based on device. Can u check if that helps rather than using `text-14`?